c2corg_ui/static/js/card.js

```diff
@@ -130,6 +130,34 @@ app.CardController.prototype.translate = function(str) {
 
 
 /**
+ * Show only one of the area types, the first that is available:
+ * 1) range 2) admin limits 3) country
+ * @param {?Array<Object>} areas
+ * @return {string | null}
+ * @export
+ */
+app.CardController.prototype.showArea = function(areas) {
+  if (!app.utils.isTopoguide(window.location.pathname.substring(1)) && areas) {
+    // the areas often come in different orders within 3 area objects
+    var orderedAreas = {};
+
+    for (var i = 0; i < areas.length; i++) {
+      orderedAreas[ areas[i]['area_type']] = areas[i]['locales'][0]['title'];
```

Comment:
This means that only one region of a given type will be listed, doesn't it?
But we have routes or outings that span over several regions of the same type.

What about
```
var orderedAreas = {'range': [], 'admin_limits': [], 'country': []};
for (var i = 0; i < areas.length; i++) {
  var type = areas[i]['area_type'];
  orderedAreas[type].push(areas[i]['locales'][0]['title']);
}
for (var type in orderedAreas) {
  if (orderedAreas[type].length) {
    return orderedAreas[type].join(' - ');
  }
}
return '';
```
?